.github/workflows/nightly.yml

```diff
@@ -5,44 +5,60 @@ on:
 name: Nightly Release
 
 jobs:
-  nightly_release:
+  create_release:
     name: Create nightly release
     if: ${{ github.repository_owner == 'input-output-hk' }}
     runs-on: ubuntu-latest
     outputs:
+      release_date: ${{ steps.version.outputs.date }}
+      version: ${{ steps.version.outputs.version }}
       upload_url: ${{ steps.create_nightly.outputs.upload_url }}
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-        with:
-          submodules: true
 
-      - name: Get version
+      - name: Get version from jormungandr/Cargo.toml
         id: version
+        shell: python
         run: |
-          export VERSION=`cat ./jormungandr/Cargo.toml | grep "version" | head -n 1 | awk '{print $3}' | cut -d "\"" -f 2 `
-          echo ::set-output name=version::$VERSION
+          from __future__ import print_function
+          import json, re
+          from datetime import date
+          from subprocess import Popen, PIPE
+          p = Popen(
+            'cargo read-manifest --manifest-path jormungandr/Cargo.toml',
+            shell=True,
+            stdout=PIPE
+          )
+          d = json.load(p.stdout)
```

Comment:
[`subprocess`](https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module) would be better

Reply:
It is `subprocess.Popen` :smile:

Reply:
Yeah, I was thinking on `python3` and got confused.